lib/Migration/Version20002Date20250612140256.php

```diff
@@ -21,7 +21,7 @@
 /**
  * Adds root_id and options to the group folders table
  */
-class Version20001Date20250612140256 extends SimpleMigrationStep {
+class Version20002Date20250612140256 extends SimpleMigrationStep {
```

Comment:
Since this might have caused the issues like https://github.com/nextcloud/groupfolders/issues/4003, it's probably better to use https://github.com/nextcloud/groupfolders/pull/4022/ for this